lib/receivex/adapter.ex

```diff
@@ -1,9 +1,12 @@
 defmodule Receivex.Adapter do
   @moduledoc """
-  Behaviour for handling webhooks for different providers
+  Behaviour for handling webhooks
   """
   @callback handle_webhook(conn :: Plug.Conn.t(), handler :: Atom.t(), opts :: []) ::
-              {:ok, conn :: Plug.Conn.t()} | {:error, conn :: Plug.Conn.t(), String.t()}
+              {:ok, conn :: Plug.Conn.t()}
+              | {:ok, conn :: Plug.Conn.t(), map()}
+              | {:error, conn :: Plug.Conn.t(), String.t()}
```

Comment:
I'm not sure if this is the best way to handle multiple possible tuple response. Feedback appreciated.